metrics/grafana/strimzi-kafka-connect.json

```diff
@@ -0,0 +1,746 @@
+{
```

Comment:
in the Strimzi project, there is just one JSON for Kafka Connect, we should have just one here as well because it works with both Kafka Connect and Kafka Connect S2I.